### ISSUE TITLE
highlightNeighbour: should be named highlightNeighbors:

### ISSUE THREAD

**Issue by bergel** [moosetechnology/Moose#1161]
Neighbour is singular, and in British English.

**Comment by bergel** [comment]
Moreover, 
```
b := RTMondrian new.
b interaction highlightNeighbour: [ :v | Array with: v + 1 ].
b nodes: #(1 2 2).

b
```
Just highlight one element. It should highlight the two elements

**Comment by bergel** [comment]
In Roassal2-AlexandreBergel.1541

- highlightNeighbour: should be named highlightNeighbors:
- Allow for multiple elements with the same model to be highlighted.

Fixed Bug #1161
